packages/table/src/react/__specs__/__snapshots__/storyshots.spec.js.snap

```diff
@@ -1627,6 +1631,7 @@ exports[`Storyshots drawer single 1`] = `
       data-css-1i2htk9=""
     >
       <div
+        aria-hidden={true}
```

Comment:
These are the snapshot updates (broken in master) that I've been asking Edward about because of his Drawer work.  I'm not 100% sure that the addition of `aria-hidden` here is correct.  Are you?